basil/TL/SensirionSensorBridge.py

```diff
@@ -0,0 +1,90 @@
+#
+# ------------------------------------------------------------
+# Copyright (c) All rights reserved
+# SiLab, Institute of Physics, University of Bonn
+# ------------------------------------------------------------
+#
+
+import logging
+import array
+
+from basil.TL.TransferLayer import TransferLayer
+
+logger = logging.getLogger(__name__)
+
+try:
+    from sensirion_shdlc_driver import ShdlcSerialPort, ShdlcConnection
+    from sensirion_shdlc_sensorbridge import SensorBridgePort, SensorBridgeShdlcDevice
+    from sensirion_shdlc_sensorbridge.device_errors import SensorBridgeI2cTimeoutError
+    TimeoutError = SensorBridgeI2cTimeoutError
+except ModuleNotFoundError:
+    logger.error("You have to install the package 'sensirion_shdlc_sensorbridge' to use this device.")
+    raise
```

Comment:
Why are you handling import errors manually?

Reply:
I did not want to have this pakage as a dependecy for basil. I just wanted to message which package to install if required.